src/radio/radio-group.component.ts

```diff
@@ -239,10 +248,13 @@ export class RadioGroup implements AfterContentInit, AfterViewInit, ControlValue
 	/**
 	 * Synchronizes the names of the radio items with the name of the `RadioGroup`.
 	 */
-	updateRadioNames() {
+	updateRadios() {
```

Comment:
Why the name change? Technically, it's a public function so technically, this is a breaking change...

If this is something we actually want, we go the deprecate and substitute route.